Makefile

```diff
@@ -16,7 +16,7 @@ endif
 SUBDIRS=kafka-agent mirror-maker-agent tracing-agent crd-annotations test crd-generator api api-conversion mockkube certificate-manager operator-common config-model config-model-generator cluster-operator topic-operator user-operator kafka-init docker-images packaging/helm-charts/helm3 packaging/install packaging/examples
 DOCKER_TARGETS=docker_build docker_push docker_tag
 
-all: prerequisites_check $(SUBDIRS) crd_install helm_install shellcheck docu_versions docu_check
+all: prerequisites_check publish_check $(SUBDIRS) crd_install helm_install shellcheck docu_versions docu_check
```

Comment:
Out of curiosity ... why do you call it publish_check?

Reply:
This was more of a placeholder, I wasn't 100% sure what to call it, i put publish as these are like our 'published' materials, maybe `release_files_check` might be better?
Is there a specific word for what you call the public facing, release versioned files for helm-charts/install/examples?

Reply:
I think `release_files_check` sounds good to me. I think we can use it. _release files_ is what I would normally call it I guess.